docs/spec_draft.md

```diff
@@ -585,9 +617,9 @@ returned.
 
 ### Outputs
 
-| Name      | Type                                             |
-|-----------|--------------------------------------------------|
-| `results` | variadic number of tensors of any supported type |
+| Name      | Type                                                      |
+|-----------|-----------------------------------------------------------|
+| `results` | variadic number of tensors of token or any supported type |
```

Comment:
"variadic number of tensors of any supported type or tokens"? `HLO_TensorOrToken` means `AnyTypeOf<[HLO_Tensor, HLO_Token]>`.